@xen-orchestra/lite/src/components/Tree.tsx

```diff
@@ -113,24 +109,14 @@ const renderItem = ({ children, id, label, to, tooltip }: ItemType) => {
 }
 
 const Tree = withState<State, Props, Effects, Computed, ParentState, ParentEffects>(
-  {
-    initialState: ({ defaultSelectedNodes }) => ({
-      selectedNodes: defaultSelectedNodes === undefined ? [] : defaultSelectedNodes,
-    }),
-    effects: {
-      setSelectedNodeIds: function (event, nodeIds) {
-        this.state.selectedNodes = nodeIds
-      },
-    },
```

Comment:
- Why? TreeView will be a controlled component when using `multiSelect`.
- We shouldn't fix this by an array of `undefined`.

Reply:
Maybe it can be possible to use an empty string as selected ( or an array of empty string)  ?